tests/test_client.py

```diff
@@ -393,3 +393,33 @@ def test_client_errors(self):
         err = oqe.ConnectionError("oh no")
         assert err.args[0] == "oh no"
         assert err.err == "oh no"
+
+    @mock.patch("openqa_client.client.OpenQA_Client.openqa_request", autospec=True)
+    def test_get_latest_build(self, fakerequest):
+        client = oqc.OpenQA_Client()
+        with pytest.raises(TypeError):
+            client.get_latest_build()
```

Comment:
TBH it does not make sense to me too , but I was just trying to following same logic used by other tests here . Should I also delete same pattern in lines   368-369 ?
```
with pytest.raises(TypeError):
            client.get_jobs()
```

Reply:
We have that there because `get_jobs` is kinda 'special'. All its arguments are optional. Python itself won't raise `TypeError` if you do `get_jobs()`. But we *actually* need you to pass *either* the `jobs` arg or the `build` arg, and if you don't pass either of those, *we* raise `TypeError` - that's line 364 in client.py. Because that code is in our library it needs test coverage.

I actually hate this and wish I hadn't built it that way, but we're stuck with it now. :P

Reply:
done

Reply:
er...but...you didn't remove the check?